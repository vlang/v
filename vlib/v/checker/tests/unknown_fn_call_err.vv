module main

fn main() {
	C.printf(c'%f\n', sin(2))
}
