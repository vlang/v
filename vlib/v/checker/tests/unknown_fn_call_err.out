vlib/v/checker/tests/unknown_fn_call_err.vv:4:20: error: unknown function: sin
    2 | 
    3 | fn main() {
    4 |     C.printf(c'%f\n', sin(2))
      |                       ~~~~~~
    5 | }
